docs/pages/component/props.mdx

```diff
@@ -157,6 +158,7 @@ controlsStyles={{
   hideSeekBar: false,
   hideDuration: false,
   seekIncrementMS: 10000,
+  hideNavigationBarOnFullScreenMode: false,
```

Comment:
Do we really need this prop? I think people expect `fullscreen` already include this.
I think giving freedom to features that are not closely related to video would increase the number of management points and leave a lot of room for side effects.🤔

Reply:
@freeboub @KrzysztofMoch 
WDYT?